### PR TITLE
Use LabIcon in code-snippet package

### LINKED ISSUE #430: Update arrow icons in code snippets
Currently the arrows for expanding a code snippet in the UI are `v` for unexpanded and `^` for expanded. This usage matches the UX for opening and closing an expandable section, as we do in our error dialog, because the down arrow says "click for more info" and the up arrow says "hide more info." For code snippets the expanded section is more like subsection. As such it would make more sense for unexpanded content to use `>` and expanded content to use `v` this would match how the git extension handled its expandable sections.

### LINKED ISSUE #443: Switch to LabIcon in Code Snippets
Splitting this out of #439  since it is non-blocking to the update to 2.0

We should update code snippets to use LabIcons for all icon use similar to the rest of the packages.

### PR DESCRIPTION
**Pull request by ajbozarth**
Update the icon usage in code snippets to use LabIcons

This includes:
- Moving the code-snippet logo into application
- tweaking the svg viewBox to remove excess space around the icon
which make it properly fill the width off the icon in the css
- replace all icons set with css with LabIcon instances
- some minor icon css size tweaking

Fixes #443
Fixes #430

 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: packages/application/style/code-snippet.svg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<!DOCTYPE svg  PUBLIC '-//W3C//DTD SVG 1.1//EN'  'http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd'>
+<svg height="500px" id="Layer_1" version="1.1" viewBox="125 150 250 200" width="500px" xml:space="preserve" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <g>
+        <g>
+            <polygon fill="#231F20" points="195.568,185.811 142.681,250.173 195.568,314.534 208.077,293.962 172.44,250.173      208.077,206.384    "/>
+            <polygon fill="#231F20" points="228.707,313 251.922,313 288.816,187 265.6,187    "/>
+            <polygon fill="#231F20" points="303.876,185.81 291.37,206.384 327.005,250.173 291.37,293.961 303.876,314.535 356.765,250.173         "/>
```
Comment (vabarbosa) [inline packages/application/style/code-snippet.svg:8]: not a big deal, but perhaps remove the extra spaces in the `points`
```suggestion
            <polygon fill="#231F20" points="195.568,185.811 142.681,250.173 195.568,314.534 208.077,293.962 172.44,250.173 208.077,206.384"/>
            <polygon fill="#231F20" points="228.707,313 251.922,313 288.816,187 265.6,187"/>
            <polygon fill="#231F20" points="303.876,185.81 291.37,206.384 327.005,250.173 291.37,293.961 303.876,314.535 356.765,250.173"/>
```

Comment (ajbozarth) [reply packages/application/style/code-snippet.svg]: I actually just copied the svg file "as is" and then added some newlines for readability before I edited the viewbox. I committed your suggestion though cause I see no reason not to.

**Comment by ajbozarth** [comment]
The tests actually consistently fail locally as well, I'm taking a look to figure out why

**Comment by ajbozarth** [comment]
I've tracked down the cause: It's the same issue as has been raised in the git extension https://github.com/jupyterlab/jupyterlab-git/issues/604 
I'm testing a work around but given this is now hitting us twice I'm going to open a PR on lab with my idea I suggested in the git pr

**Comment by ajbozarth** [comment]
I've pushed a workaround fix for the tests and will be opening a PR on lab to fix it there since it's also happening to jupyterlab-git

**Pull request merged**